cdc_rsync_server/main.cc

```diff
@@ -60,10 +70,19 @@ ServerExitCode GetExitCode(const absl::Status& status) {
 }  // namespace cdc_ft
 
 int main(int argc, const char** argv) {
-  if (argc < 2) {
-    printf("Usage: cdc_rsync_server <port> cdc_rsync_server <size> <time> \n");
-    printf("       where <size> and <time> are the file size and modified\n");
-    printf("       timestamp (Unix epoch) of the corresponding component.\n");
+  if (argc == 2 && strcmp(argv[1], "--version") == 0) {
+    printf("%s\n", BUILD_VERSION);
+    return 0;
+  }
+
+  if (argc < 5) {
+    printf(
+        "Usage: cdc_rsync_server <port> cdc_rsync_server <size> <time> "
+        "<build_id> \n");
```

Comment:
Nit: build_id is not consistent with BUILD_VERSION, which makes it harder to understand that the two are the same. Let's be consistent and either call it build_id or build_version everywhere.